scripting/shavit-timelimit.sp

```diff
@@ -244,6 +298,58 @@ void SetLimit(int time)
 	gCV_RestartGame.IntValue = 1;
 }
 
+public void Player_Spawn(Event event, const char[] name, bool dontBroadcast)
+{
+	int client = GetClientOfUserId(event.GetInt("userid"));
+
+	if(gB_ForceMapEnd)
+	{
+		CreateTimer(1.0, CheckRemainingTime, client, TIMER_REPEAT|TIMER_FLAG_NO_MAPCHANGE);
```

Comment:
This doesn't work. You pass a client index instead of a client serial.
Also, I'd rather have a global timer instead of a timer per-player.